cloud/aws/services/ec2/instances.go

```diff
@@ -32,7 +32,7 @@ import (
 func (s *Service) InstanceIfExists(instanceID *string) (*v1alpha1.Instance, error) {
```

Comment:
Could we use just a normal string here?

Reply:
Maybe, probably? I was just making the minimal change to stop the NPE.